integration/util_test.go

```diff
@@ -0,0 +1,129 @@
+//go:build requires_docker
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/grafana/rollout-operator/integration/k3t"
+)
+
+// requireEventuallyPod runs the tests provided on the pod retrieved from the API.
+// Once a test suceeds, it's not evaluated again.
```

Comment:
[nit]
```suggestion
// Once a test succeeds, it's not evaluated again.
```

Reply:
Done in https://github.com/grafana/rollout-operator/pull/30/commits/1cb4c73912501c7d075d2490c83c92a49fccbc7c